src/components/game/block.js

```diff
@@ -1,3 +1,35 @@
+// import { initBlockArray } from "../../renderer/renderer"
```

Comment:
I guess this line should be removed if not required.
If we will need here, please mark as "TODO" or "Fixme" comment :)